x-pack/plugins/cases/public/components/all_cases/table_filters.tsx

```diff
@@ -59,47 +68,42 @@ const CasesTableFiltersComponent = ({
   availableSolutions,
   displayCreateCaseButton,
   onCreateCasePressed,
+  isLoading,
+  currentUserProfile,
 }: CasesTableFiltersProps) => {
-  const [selectedReporters, setSelectedReporters] = useState(
-    initial.reporters.map((r) => r.full_name ?? r.username ?? '')
-  );
   const [search, setSearch] = useState(initial.search);
   const [selectedTags, setSelectedTags] = useState(initial.tags);
   const [selectedOwner, setSelectedOwner] = useState(initial.owner);
+  const [selectedAssignees, setSelectedAssignees] = useState<UserProfileWithAvatar[]>([]);
+  const fetchAssignees = useRef<() => void>();
```

Comment:
Don't forget to remove the `fetchAssignees` ref. It is not needed. `queryClient.refetchQueries` is sufficient.